server/app/api.js

```diff
@@ -28,5 +29,12 @@ router.get('/post', async (req, res) => {
   res.json(await Post.find({}));
 });
 
+router.get('/broadcast', async (req, res) => {
+  await Broadcast.find({}).sort('-date').exec(function(err, broadcast) {
```

Comment:
When doing it this way, the use of `async/await` is unneeded. When using `async/await`, it would look something like this:

```javascript
try {
  res.json(await Broadcast.find({}).sort('-date'));

catch (e) {
   // error handling
}
```